edx_proctoring/api.py

```diff
@@ -85,6 +86,7 @@ def create_exam(course_id, content_id, exam_name, time_limit_mins, due_date=None
         is_practice_exam=is_practice_exam,
         is_active=is_active,
         hide_after_due=hide_after_due,
+        backend=backend
```

Comment:
nit: use a trailing comma.  (I know you aren't a fan of super-shallow review... There will also be more substantive comments.)